linguistics/src/main/java/com/yahoo/language/process/Embedder.java

```diff
@@ -15,9 +15,22 @@
  */
 public interface Embedder {
 
+    /** Name of embedder when none is explicity given */
+    String defaultEmbedderName = "defaultEmbedder";
```

Comment:
I think we should use "id" not "name" here, and just use the id "default".

Reply:
:+1: